models/wholeBrainSeg_Large_UNEST_segmentation/docs/README.md

```diff
@@ -0,0 +1,148 @@
+# Description
+Detailed whole brain segmentation is an essential quantitative technique in medical image analysis, which provides a non-invasive way of measuring brain regions from a clinical acquired structural magnetic resonance imaging (MRI).
+
+We provide the pre-trained model for inferencing whole brain segmentation with 133 structures.
+
+A tutorial and release of model for whole brain segmentation using the 3D transformer-based segmentation model UNEST.
+
+Authors:
+Xin Yu (xin.yu@vanderbilt.edu) (Primary)
+
+Yinchi Zhou (yinchi.zhou@vanderbilt.edu) | Yucheng Tang (yuchengt@nvidia.com)
+
+<p align="center">
+-------------------------------------------------------------------------------------
+</p>
+
+<!-- ![](./demo.png) <br> -->
+<p align="center">
+<img src="./demo.png" width="800" height="456">
```

Comment:
To insert image in the markdown document, you can refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/docs/source/highlights.md?plain=1#L87

Thanks.

Reply:
Sure, changed back to the format.

Reply:
I see you still have the HTML code here?

Thanks.